networkx/algorithms/tests/test_graph_hashing.py

```diff
@@ -1,42 +1,624 @@
+import pytest
 import networkx as nx
 
 
-def test_empty_graph_hash():
-    G1 = nx.empty_graph()
-    G2 = nx.empty_graph()
+def _relabel_graph(G, f=lambda x: -1 * x):
+    """
+    relabels graph nodes by a given relabeling function on integers.
+    By default it flips the sign of the labels (Note this is a self-inverse).
+    Returns the relabeled graph, with the original left intact.
+    """
+    return nx.relabel_nodes(G, {u: f(u) for u in G.nodes()})
 
-    h1 = nx.weisfeiler_lehman_graph_hash(G1)
-    h2 = nx.weisfeiler_lehman_graph_hash(G2)
 
-    assert h1 == h2
+def _relabel_dict(dic, f=lambda x: -1 * x):
```

Comment:
Similar comment for this `_relabel_dict` function. Input `f` is not used.  Maybe simple enough to just inline this code.

Reply:
changed to inline